profile/README.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to Open Horizon 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="https://github.com/Ayush7614/.github/blob/master/profile/open-horizon-color.png" width="55%" />
+</p>
+
+<p align="center">
+
+[![Slack](https://img.shields.io/badge/slack-@lf_edge--community-yellow.svg)](https://lfedge.slack.com/)
+[![Twitter](https://img.shields.io/badge/twitter-@lf_edge--twitter-9cf.svg)](https://twitter.com/lf_edge)
+[![YouTube](https://img.shields.io/badge/youtube-@lf_edge--youtube-red.svg)](https://www.youtube.com/channel/UCY7H1oSt8gvXNdXH9wrNq5Q)
+<a href="https://chat.lfx.linuxfoundation.org/#/welcome" alt="Element">
+<img src="https://img.shields.io/badge/Element-@element.svg?logo=element" /></a>
+</p>
+
+[Open Horizon](https://www.lfedge.org/projects/openhorizon/) is a platform for managing the service software lifecycle of containerized workloads and related machine learning assets. It enables autonomous management of applications deployed to distributed webscale fleets of edge computing nodes and devices without requiring on-premise administrators.
+
+### 🏂 Haven't tried Open Horizon yet ?
+Here are few things which we have developed in last couple of years.
```

Comment:
Instead of "Here are few things which we have developed in last couple of years." let's say:
"Here are some steps to learn and use the Open Horizon component services."
1. [Install the Management Hub Services](https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md#-deploy-all-in-1-horizon-management-hub-agent-and-cli) on an x86 machine or VM using our easy all-in-one script.
2. [Install the Agent](https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md#adding-more-edge-devices) on almost any single-board computer (SBC), desktop, laptop, or Kubernetes cluster.
3. Publish and use [our example services](https://github.com/open-horizon/examples/tree/master/edge/services).
4. [Create and publish your own services](https://github.com/open-horizon-services/Getting-Started#contributing-a-new-microservice).